tests/test_flake8_trio.py

```diff
@@ -25,13 +25,8 @@
 
 from flake8_trio import ERROR_CLASSES, Error, Plugin, Statement
 
-# TODO: Move test_eval files into a separate directory
-trio_test_files_regex = re.compile(r"trio\d.*.py")
-
-test_files: list[tuple[str, str]] = sorted(
-    (os.path.splitext(f)[0].upper(), f)
-    for f in os.listdir("tests")
-    if re.match(trio_test_files_regex, f)
+test_files: list[tuple[str, Path]] = sorted(
+    (f.stem.upper(), f) for f in (Path(__file__).parent / "eval_files").iterdir()
 )
```

Comment:
No need for a regex, and using `Path` functions to do `os.path.splitext` (`.stem`) and `os.listdir` (`.iterdir()`).
It now also runs regardless of your current working directory, while previously it required being in the project root.